src/graphql/main/schema.graphql

```diff
@@ -974,6 +998,11 @@ type Query {
     @deprecated(
       reason: "TODO: remove. we don't need a non authenticated version of this query. the users can only do the query while authenticated"
     )
+
+  """
+  Returns 1 Sat and 1 Usd Cent price for the given currency
+  """
+  realtimePrice(currency: DisplayCurrency! = "USD"): RealtimePrice
```

Comment:
this would be use by `galoy-pay`?

Reply:
the `!` means mandatory? I'm confused the combination of this + a default argument

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/2298/commits/46de91a0de0d3b19db1751129dfd056c7167a506